lib/circuitry/subscriber.rb

```diff
@@ -81,7 +87,15 @@ def lock=(value)
     private
 
     def receive_messages(&block)
-      response = sqs.receive_message(queue, 'MaxNumberOfMessages' => batch_size, 'WaitTimeSeconds' => wait_time)
+      response = nil
+
+      begin
+        response = sqs.receive_message(queue, 'MaxNumberOfMessages' => batch_size, 'WaitTimeSeconds' => wait_time)
+      rescue *TEMPORARY_ERRORS => e
+        logger.info("Temporary issue connecting to SQS: #{e.message}")
+        return
+      end
+
       messages = response.body['Message']
```

Comment:
is response.body going to throw and error if response is nil?

should this be `response.body['Message'] if response`